configserver/src/main/java/com/yahoo/vespa/config/server/maintenance/SessionsMaintainer.java

```diff
@@ -25,6 +26,9 @@
 
     @Override
     protected boolean maintain() {
+        if (iteration % 10 == 0)
+            log.log(LogLevel.INFO, () -> "Running " + SessionsMaintainer.class.getSimpleName());
```

Comment:
Consider including the iteration in the message, to make it clearer for the reader of the message it is actually running more often

Reply:
Good point, done